pkg/tecdsa/gen/pb/preparams.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+option go_package = "pb";
+package tecdsa;
+
+message PreParams {
+  LocalPreParams data = 1;
+}
+
+message LocalPreParams {
+  // PrivateKey is not present for pre-params generated before DKG and is
+  // filled when DKG finishes work. That being said, this message will be
+  // empty in PreParams message and non-empty in PrivateKeyShare message
+  // (see key.proto).
+  message PrivateKey {
+    bytes publicKey = 1;
+    bytes lambdaN = 2;
+    bytes phiN = 3;
+  }
```

Comment:
In 9c62fddb2f6d392bd614d204249d1d06b55fb3ff I promoted `LocalPreParams` to be defined outside of `LocalPartySaveData`. With this change, in 78dc79f16ff66dbd503c2b152b6d775016fcbc0b, I could move `LocalPreParams` message to a separate file. This message is used by `LocalPartySaveData `  and by `PreParams` that will be saved on disk.

I am struggling with `PrivateKey` part: it is empty for `PreParams` (before DKG) and is filled for `LocalPartySaveData`. We can leave it as-is or:
1. Revert 9c62fddb2f6d392bd614d204249d1d06b55fb3ff, keep `LocalPreParams` as internal for `LocalPartySaveData`
2. Introduce separate `LocalPreParams` without `PrivateKey` and use it as an internal message of `PreParams`.

@lukasz-zimnoch thoughts on that?

Reply:
I'd vote for separate protobuf definitions for both cases. My reasoning: in this case, a little duplication seems to be a much smaller problem than introducing the additional complexity related to a field that can be in different states depending on the usage context. Protobuf definitions are meant to describe the shape of data and I think we should not have any details regarding how those data are processed here (i.e. this field should be empty before DKG but filled after). Additionally, the proposed structure can be error-prone, especially regarding the marshaling code. This is why, I think, duplication is better here, especially given the fact those data structures are not supposed to be changed very often (if at all).

Reply:
Done in https://github.com/keep-network/keep-core/pull/3168/commits/8ff42ee9956273dc3cd54f429dbeb9ce0eba8e91.